Shaders/RHI/TrianglesConstantBuffer.azsl

```diff
@@ -21,14 +21,32 @@
 
 #endif
 
+
+#if AZ_TRAIT_CONSTANT_BUFFER_ALIGNMENT <= 64
```

Comment:
Where is AZ_TRAIT_CONSTANT_BUFFER_ALIGNMENT defined?

Reply:
They are in Gems/Atom/Asset/Shader/Code/AZSL/Platform/Android/Vulkan/AzslcHeader.azsli in ASV